### DISCUSSION TITLE
[Feature] Moments Notifications

### DISCUSSION THREAD

**Discussion by andriykrefer** [immich-app/immich#11014] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

One of the coolest features of the GPhotos is its "memories" features and the fact that it display a notification on your phone to check out relevant moments today. Immich has a similar feature called "moments". Although it not as powerful, I think it is already a step in the right direction and this discussion belongs to a dedicated FR.

My feature request for now is to implement the notification part. If a moment is available at the current date, send a phone notification so the user can check it out. Otherwise, it is kind of impractical to check the app every day to see if there is a memorie available.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3981
